Moonlight macOS/Utility/keyboardTranslation.m

```diff
@@ -0,0 +1,163 @@
+//
+//  keyboardTranslation.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 10.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "keyboardTranslation.h"
+#import <Limelight.h>
+
+CGKeyCode modifierKeyFromEvent(int keyModifier) {
+    switch (keyModifier) {
+        case 131330:
+            return 0xA0; //LSHIFT
+            //case 131332:              TODO: This will lockup the modifiers
```

Comment:
What's the TODO mean exactly?

Reply:
There seems to be a difference in the handling of keyUp-Events for the right side modifiers, this lead to a bug where the key got "stuck", i have now evaded this by using the keyModifier const's. It seems that there are no separate const's for the left and right side modifiers.